upgrade/scripts/upgrade/prepare-assets.sh

```diff
@@ -26,6 +26,7 @@
 set -e
 locOfScript=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 . ${locOfScript}/../common/upgrade-state.sh
+CSM_ARTI_DIR="not_set"
```

Comment:
Why not instead set the variable correctly and then move the call to include ncn-common.sh until after this variable is set? I'll make a PR to your PR to show you what I mean.

Reply:
See what you think of this: https://github.com/Cray-HPE/docs-csm/pull/2460

I think the dummy value would work, but it seems cleaner to me to just set the correct value